src/models/simulation-magnet.ts

```diff
@@ -10,6 +10,16 @@ export type PolarityType = typeof PolarityTypeEnum.Type;
 export const CoilPolarityTypeEnum = types.enumeration("type", ["plus-minus", "off", "minus-plus"]);
 export type CoilPolarityType = typeof CoilPolarityTypeEnum.Type;
 
+export const kCoilStrengthWeak = .5;
+export const kCoilStrengthMedium = .75;
+export const kCoilStrengthStrong = 1;
+export const kBarStrengthWeak = .2;
+export const kBarStrengthMedium = .6;
+export const kBarStrengthStrong = 1;
+export const kCurrentStrengthWeak = .5;
+export const kCurrentStrengthMedium = .75;
+export const kCurrentStrengthStrong = 1;
```

Comment:
Ideally these constants would get used in the `strength-panel` too, just so they never become de-synchronized

Reply:
Ah, of course.